api/src/backend/api/models.py

```diff
@@ -49,6 +50,8 @@
 # Convert Prowler Severity enum to Django TextChoices
 SeverityChoices = enum_to_choices(Severity)
 
+logger = logging.getLogger(__name__)
```

Comment:
We shouldn't need custom logging at model level, but in any case, use the API`s custom logger, not `__name__`.